operations/helm/policies/policies.rego

```diff
@@ -28,11 +34,57 @@ pod_template_has_labels(template, labels) {
 	}
 }
 
-deny["TopologySpreadConstraints use labels not presnet on pods"] {
-	can_use_topology_spread_constraints(input[i].contents.kind)
+deny[msg] {
+	obj := input[i].contents
+	msg := sprintf("TopologySpreadConstraints use labels not presnet on pods: %v", [obj])
+	can_use_topology_spread_constraints(obj.kind)
 
-	pod_template := input[i].contents.spec.template
-	topology_spread_constraints_labels := input[i].contents.spec.template.spec.topologySpreadConstraints[j].labelSelector.matchLabels
+	pod_template := obj.spec.template
+	topology_spread_constraints_labels := obj.spec.template.spec.topologySpreadConstraints[j].labelSelector.matchLabels
 
 	not pod_template_has_labels(pod_template, topology_spread_constraints_labels)
 }
+
+is_mimir_or_gem_image(image) {
+	startswith(image, "grafana/mimir")
+}
+
+is_mimir_or_gem_image(image) {
+	startswith(image, "grafana/enterprise-metrics")
+}
+
+deny[msg] {
+	obj := input[i].contents
+	msg := sprintf("Mimir/GEM containers doesn't have the restricted security context: %v", [obj])
```

Comment:
```suggestion
	msg := sprintf("Mimir or GEM containers do not have the restricted security context: %v", [obj])
```

Reply:
Check whether or not this should be singular or plural and adjust grammar as needed.

Also change any like occurrences.